### DISCUSSION TITLE
Welcome to awesome-brain-manager Discussions!

### DISCUSSION THREAD

**Discussion by JuckZ** [JuckZ/awesome-brain-manager#24] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ruamsx** [comment]
## 关于提醒功能

目前 obsidian 有 Reminder 插件可以做到在 obsidian 内实现定时提醒功能，但是没法将提醒发送到手机端。
不知道有没有什么办法能将 obsidian 内的任务到期提醒发送到手机。

**Reply by JuckZ** [reply]
我之前使用的是ntfy，我把它整合在这个插件里面了，但是需要实现你所说的效果还需要一点点改动，接下来我会去实现这个特性
